app/src/main/java/com/atomicrobot/carbon/StartActivity.kt

```diff
@@ -3,27 +3,34 @@ package com.atomicrobot.carbon
 import android.content.Intent
 import android.net.Uri
 import android.os.Bundle
+import androidx.activity.compose.setContent
 import androidx.activity.viewModels
 import com.atomicrobot.carbon.ui.BaseActivity
-import dagger.hilt.android.AndroidEntryPoint
+import com.atomicrobot.carbon.ui.compose.MainNavigation
 import com.atomicrobot.carbon.ui.splash.SplashViewModel
+import com.atomicrobot.carbon.ui.theme.CarbonAndroidTheme
+import dagger.hilt.android.AndroidEntryPoint
 import timber.log.Timber
 
 @AndroidEntryPoint
 class StartActivity : BaseActivity() {
     private val splashViewModel: SplashViewModel by viewModels()
-    public override fun onCreate(savedInstanceState: Bundle?) {
+    override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        setContentView(R.layout.activity_start)
-
+        setContent {
+            CarbonAndroidTheme {
+                MainNavigation(true)
```

Comment:
This should be passing in the return value from `handleIntent()`

Reply:
nice catch